library/bz2.po

```diff
@@ -119,6 +147,9 @@ msgid ""
 "file directly. Otherwise, *filename* should be a :term:`file object`, which "
 "will be used to read or write the compressed data."
 msgstr ""
+"Si *filename* est un objet :class:`str` ou :class:`bytes`, ouvre le nom de "
+"fichier directement. Autrement, *filename* devrait être un :term:`file "
```

Comment:
```suggestion
"fichier directement. Autrement, *filename* doit être un :term:`file "
```

Reply:
Depuis quand "should be" signifie "doit être" et non pas "devrait être"…. ?
Je ne valide pas cette correction

Reply:
Depuis que les anglais ont des _modaux_.

Reply:
Là, malheuresement, moi pas comprendre !

Reply:
Pour la traduction de documentation technique, une lecture intéressante et malheureusement retirée par l'éditeur est donnée en premier lien de la page du [guide stylistique pour la traduction](https://github.com/mozfr/besogne/wiki/Guide-stylistique-pour-la-traduction) de Mozilla France.